includes/filerepo/backend/SwiftFileBackend.php

```diff
@@ -571,7 +573,15 @@ protected function addMissingMetadata( CF_Object $obj, $path ) {
 			return true; //nothing to do
 		}
 		wfProfileIn( __METHOD__ );
-		trigger_error( "$path was not stored with SHA-1 metadata.", E_USER_WARNING );
+
+		WikiaLogger::instance()->warning(
+			__METHOD__ . ' - file was not stored with SHA-1 metadata',
+			[
+				'path' => $path,
+				'object' => $obj->container->name
+			]
+		);
+
 		$status = Status::newGood();
 		$scopeLockS = $this->getScopedFileLocks( array( $path ), LockManager::LOCK_UW, $status );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$scopeLockS" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)